torch/_dynamo/compiled_autograd.py

```diff
@@ -139,6 +139,23 @@ def post_hook(self, outputs, inputs, hook_id):
             self.bind_tensors_to_proxies(outputs, proxies)
         return outputs
 
+    def post_acc_grad_hook(self, inputs, hook_id):
+        assert len(inputs) == 1
+        hook = self.hooks_proxy[hook_id]
+        proxies = self.fx_tracer.create_proxy(
+            "call_function",
+            call_hook,
+            (
+                hook,
+                self.to_proxy(inputs[0])
+            ),
+            {},
+        )
+        with disable_proxy_modes_tracing():
+            inputs = maybe_clone(inputs[0])
+            self.bind_tensors_to_proxies([inputs], proxies)
+        return inputs
```

Comment:
Does this type of hook actually return anything?  The call to it in eager just ignores the result.